### PR TITLE
Remove group number from form

### PR DESCRIPTION
**Pull request by f-fromont**
- Changed form to use email instead
- removed the group number from the code

### CODE REVIEW

**Review by HiruNya — APPROVED** [review]
LGTM

File: parsons-tool-frontend/src/pages/Summary.js [hunk 1]
```diff
@@ -43,8 +41,7 @@ function Summary() {
             >
               https://docs.google.com/forms/d/e/1FAIpQLSfh2QhZeWG0AgIUqDq2idDSYvOttxHLz8vhIXOqD3EQ5gzONg/viewform?usp=sf_link
             </a>
-            <br />
-            <br /> If you do decide to fill it out, for the <b>Group</b> field, please enter the number <b>"{group}"</b>
+            <br />{' '}
```
Comment (HiruNya) [inline parsons-tool-frontend/src/pages/Summary.js:44]: Why do we need the `{' '}` - can't we just get rid of that completely?

**Review by HiruNya — DISMISSED** [review]
LGTM - have one comment

File: parsons-tool-frontend/src/pages/Summary.js (hunk 1 above)
Comment (f-fromont) [reply parsons-tool-frontend/src/pages/Summary.js:44]: I think that was added by format on save

**Review by HiruNya — APPROVED** [review]
LGTM

**Pull request merged**